templates/index.ts

```diff
@@ -16,7 +16,7 @@ export function doStartThings() {
     );
 }
 
-export function doStopThings(api: typeof theia) {
+export function stop(api: typeof theia) {
```

Comment:
I think we are not sending as well theia object on stop (I forget to remove it)
it can be fixed later as well